### PR TITLE
Power module bugfix

### PR DESCRIPTION
**Pull request by tnelson-integral**
roundup def bugfix for Issue #35.

### CODE REVIEW

File: seat/modules/power_module.py [hunk 1]
```diff
@@ -320,8 +320,23 @@ def reset_bc_data_order(bc_data):
...
+    #Round up to the nearest place
+    # +decimal = round up to the defined decimal place
+    # -decimal = round up to the nearest 10
+    # e.g. for value 100.12345, decimal -3 = 1000, decimal 3 = 100.124
+    decimal = float(decimal)
+    if decimal >= 0:
+        rounded = np.ceil(x * 10**decimal) / 10**decimal
+    else:
+        rounded = np.ceil(x / 10**(-1*decimal)) * 10**(-1*decimal)
+    return rounded
```
Comment (ssolson) [inline seat/modules/power_module.py:333]: Do we need this? I don't see this function ever called in the code.

File: seat/modules/power_module.py [hunk 2]
```diff
@@ -320,8 +320,23 @@ def reset_bc_data_order(bc_data):
...
+
+def roundup(x, val=2):
+    rounded = np.ceil(x)
+    while np.mod(rounded, val) != 0:
+        rounded += 1
+    return rounded
```
Comment (ssolson) [inline seat/modules/power_module.py:339]: What about the following which eliminates the need for a while loop?
```
def roundup(x, val=2):
    return np.ceil(x / val) * val

# Test
print(roundup(7, 2))  # Output: 8.0
print(roundup(7, 5))  # Output: 10.0
```

**Review by ssolson — COMMENTED** [review]
Hey Tim could you take a look at these comments?

File: seat/modules/power_module.py (hunk 2 above)
Comment (tnelson-integral) [reply seat/modules/power_module.py:339]: @ssolson Your code works great, feel free to switch it to that. roundup_placevalue def can be removed. I didn't remove it after testing which roundup should have been used for plotting. Let me know if you want me to make the changes or if you plan to.

Comment (ssolson) [reply seat/modules/power_module.py:339]: Tim could you review and approve 
https://github.com/IntegralEnvision/seat-qgis-plugin/pull/6
when you get a chance?

Comment (tnelson-integral) [reply seat/modules/power_module.py]: @ssolson Looks good. I merged your commits.

**Pull request merged**